script/RumpelConfig.sol

```diff
@@ -72,6 +74,10 @@ library RumpelConfig {
     address public constant MAINNET_FLUID_VAULT_SUSDE_USDC = 0x3996464c0fCCa8183e13ea5E5e74375e2c8744Dd;
     address public constant MAINNET_FLUID_VAULT_SUSDE_USDT = 0xBc345229C1b52e4c30530C614BB487323BA38Da5;
     address public constant MAINNET_FLUID_VAULT_SUSDE_GHO = 0x2F3780e21cAba1bEdFB24E37C97917def304dFFA;
+    address public constant MAINNET_FLUID_VAULT_DEX_SUSDE_USDT_USDT = 0x7503b58Bb29937e7E2980f70D3FD021B7ebeA6d0; // sUSDe-USDT/USDT
+    address public constant MAINNET_FLUID_VAULT_SUSDE_DEX_USDC_USDT = 0xe210d8ded13Abe836a10E8Aa956dd424658d0034; // sUSDe/USDC-USDT
```

Comment:
Is `dex` after SUSDE here on purpose?

Reply:
Yes! what follows dex is always the two assets that are together (so this is the one strat for which the debt assets are dex). Maybe still a bit wonky, but follows the convention fluid is using in their repo